skycultures/chinese_song_dynasty/description.md

```diff
@@ -24,29 +24,29 @@ Below is *Xinyixiangfayao Star Map* from Su Song(苏颂, 1020-1101 AD).
 
 ### Restore Sky Map
 
-The sky map of the Song dynasty is typical of the Chinese medieval sky map. In 1052 AD, the Song Dynasty official conducted accurate stellar measurements, leaving a lot of data.
+The Imperial Observatory of the Song Dynasty conducted several large-scale stellar observations with remarkable accuracy, surpassing all previous eras in precision. The data from 1052 AD is the most complete set that survives today. Based on these records, two significant star charts were produced: the *Xinyixiangfayao Star Map* (1088 AD) and the *Suzhou Stone Star Map* (1247 AD).
 
-Pan Nai(潘鼐) collated these astronomical observations data and based on the *Xinyixiangfayao Star Map*, he restored most of the Song Dynasty's Xingguan in his masterpiece *The History of Stellar Observation in China*, which is the most important reference book. Qi Rui (齐锐) and Wan Haoyi (万昊宜) also restored the Xingguans of the Song Dynasty in the book *Roving China Heavens*, which is also an important reference book.
+The historian of astronomy Pan Nai (潘鼐) systematically compiled this observational data into the *Huangyou Star Catalog*. By cross-referencing it with the *Xinyixiangfayao Star Map* in his work *The History of Stellar Observation in China*, he reconstructed most of the Song Dynasty constellations, serving as the primary reference for this project.
 
-Due to the limitation of observation conditions in Song Dynasty, combined with instruments, circulated errors and other reasons, different scholars restored the sky map is somewhat different.
+In their work *Roving China Heavens*, Qi Rui (齐锐) and Wan Haoyi (万昊宜) also accomplished a restoration based on the *Suzhou Star Map*, making it another essential reference.
```

Comment:
Again, add year or better add this work to the references (if not yet included) and cite with reference number.